pkg/network/tracer/process_cache.go

```diff
@@ -59,10 +52,6 @@ type processCache struct {
 	cacheByPid map[uint32]processList
```

Comment:
why not using a sync.Map here instead of a cache mutex ?

Reply:
The mutex is protecting more than the map here.